src/index.js

```diff
@@ -1,10 +1,16 @@
 import mongoose from 'mongoose'
+import dotenv from 'dotenv'
+
+import User from './models/User.js'
+import Account from './models/Account.js'
 
 import routes from './routes/index.js'
 
+dotenv.config({ path: '../.env' })
```

Comment:
It's also ok, but you can also use the packagejson solution we discussed today. Right?